server/src/repositories/search.repository.ts

```diff
@@ -193,6 +194,26 @@ export class SearchRepository {
     return paginationHelper(items, pagination.size);
   }
 
+  @GenerateSql({
+    params: [
+      { page: 1, size: 100 },
+      {
+        takenAfter: DummyValue.DATE,
+        lensModel: DummyValue.STRING,
+        withStacked: true,
+        isFavorite: true,
+        userIds: [DummyValue.UUID],
+      },
```

Comment:
This doesn't look correct. The function only takes one argument.

Reply:
Probably not; it's copied from `searchMetadata` as it originally took almost the same arguments. However I'm not sure what it should be - I'm unfamiliar with this annotation and it's not obvious to me why some of the properties are included here and others are not. Could you help me understand what it should be?